pom.xml

```diff
@@ -83,7 +82,7 @@
     <dependency.commons-vfs-revision>2.2</dependency.commons-vfs-revision>
     <dependency.bean-matchers.revision>0.9</dependency.bean-matchers.revision>
     <pentaho-hadoop-shims-api.version>9.0.0.0-SNAPSHOT</pentaho-hadoop-shims-api.version>
-    <dependency.avro.revision>1.6.2</dependency.avro.revision>
+    <apache-avro.version>1.8.2</apache-avro.version>
```

Comment:
I'm not familiar with this code base. Do we need to check with anyone regarding the potential impacts of upgrading avro or are you at ease with the code base?

Reply:
@graimundo, `apache-avro` is used in the **Avro input** and **Avro output** steps in PDI. I'm comfortable with these changes and the tests it will require. Also, _pentaho-hadoop-shims_ already uses `apache-avro` 1.8.1, which should also be updated to 1.8.2.

A minor issue as in shims the apache-avro dependency has scope **provided** - https://github.com/pentaho/pentaho-hadoop-shims/blob/master/common/common-shim/pom.xml#L106.